### PR TITLE
Fix orders and projects index view; closes #163 addresses #198

### LINKED ISSUE #163: As a lender, when I click the lend button on the project index page or the tenant/project show page, I can enter in amounts greater than $10 and not amounts greater than the project price via a dollar type-in field or select box in the cart page.

### PR DESCRIPTION
**Pull request by EmilyMB**
This PR fixes a lot of the issues that came up on the review today.

Changes include:
* Only allow loans to be updated with values of $10 or more, unless project needs less than $10 (in which case the user must give the exact amount needed).
* Update projects_path to only include the projects that have active AND approved tenants AND are not fully funded. Updates Lend buttons to have include the minimum of either $25 or the remaining amount needed
* Removes before/after filters that weren't working.

NOTE: other views need to be updated still, so #198 isn't closed yet.

### CODE REVIEW

File: app/controllers/pending_loans_controller.rb [hunk 1]
```diff
@@ -20,8 +20,7 @@ def update
         pending_loan_params[:loan_dollar_amount].to_f * 100
       flash[:notice] = "Project loan amount updated"
     else
-      flash[:notice] = "Please enter a valid amount between $10
-                        & $#{pending_project_amount / 100}"
+      flash[:notice] = notice(pending_project_amount)
```
Comment (mikedao) [inline app/controllers/pending_loans_controller.rb]: Love this breaking it out.

File: app/controllers/pending_loans_controller.rb [hunk 2]
```diff
@@ -69,8 +68,21 @@ def delete_project_from_cart
     session[:pending_loan].delete(pending_loan_params[:project_id])
   end
 
-  def valid_loan_amount(max_amount)
+  def valid_loan_amount(remaining_amount)
     amount = pending_loan_params[:loan_dollar_amount].to_f
-    amount > 10 && amount <= max_amount / 100.00
```
Comment (mikedao) [inline app/controllers/pending_loans_controller.rb]: This is great how its broken out into another method.

File: app/models/loan.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@ class Loan < ActiveRecord::Base
   validates :project_id, presence: true
   validates :amount, presence: true, numericality:
     {
-      greater_than_or_equal_to: 1000,
+      greater_than_or_equal_to: 0,
```
Comment (mikedao) [inline app/models/loan.rb]: Good change.

File: app/models/order.rb [hunk 4]
```diff
@@ -29,18 +28,4 @@ def final_total
...
-    calculate_order_total
-  end
-
-  def calculate_order_total
-    if loans.present?
-      self.total_cost = loans.map(&:amount).reduce(:+)
-    else
-      self.total_cost = 0
-    end
-  end
```
Comment (mikedao) [inline app/models/order.rb]: Why were these removed? Where did the logic go?

File: app/models/tenant.rb [hunk 5]
```diff
@@ -23,4 +23,8 @@ def self.inactive
   def self.actives
     Tenant.where(active: true)
   end
+
+  def visible_to_lenders
+    active && approved
+  end
```
Comment (mikedao) [inline app/models/tenant.rb]: I really like this.

:sparkles:

**Comment by mikedao** [comment]
Great amount of tests.

File: app/models/order.rb (hunk 4 above)
Comment (EmilyMB) [inline app/models/order.rb]: I was using these with before_add and after_add filters that weren't working, so I just changed to have a method final_total instead.

Comment (mikedao) [inline app/models/order.rb]: :+1:

**Comment by mikedao** [comment]
All changes look good, great work @EmilyMB!

:sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles:

**Pull request merged**